README.md

```
@@ -67,7 +67,20 @@ The documentation is hosted on [Read the Docs](https://holidays.readthedocs.io).
+
+### holidays-ics
+
+See the [holidays-ics documentation](https://holidays.readthedocs.io/en/latest/examples/#holidays-ics-tool) for more examples.
+
+To generate a US calendar spanning the next 10 years and save it to a file (US_YYYY_YYYY.ics):
+
+```shell
+holidays-ics US --years +10
+```
+
+### web-based-holidays-ics
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)